client/my-sites/stats/site.jsx

```diff
@@ -978,6 +979,7 @@ export default connect(
 			supportsPlanUsage,
 			supportsUTMStatsFeature: supportsUTMStats,
 			supportsDevicesStatsFeature: supportsDevicesStats,
+			supportUserFeedback,
```

Comment:
Very nit-picky but we should be consistent with the naming and update this to `supportsUserFeedback` to match the others.